README.md

```diff
@@ -1,5 +1,13 @@
 # `rustc_apfloat` (Rust port of C++ `llvm::APFloat` library)
 
+## Licensing
+
+This project is licensed under the [Apache 2] with [LLVM exception] license.
+For a more complete discussion of this project's licensing, see [LICENSE-DETAILS.md](./LICENSE-DETAILS.md).
+
+[Apache 2]: https://spdx.org/licenses/Apache-2.0.html
+[LLVM Exception]: https://spdx.org/licenses/LLVM-exception.html
```

Comment:
Normally this goes near the end, but this is intentionally different due to the unique situation here, right?

Reply:
I did not put much thought into the placement of this section so I'll just move it to the bottom.